autogpt_platform/backend/backend/util/service.py

```diff
@@ -44,85 +23,26 @@
 C = TypeVar("C", bound=Callable)
 
 config = Config()
-pyro_host = config.pyro_host
-pyro_config.MAX_RETRIES = config.pyro_client_comm_retry  # type: ignore
-pyro_config.COMMTIMEOUT = config.pyro_client_comm_timeout  # type: ignore
+api_host = config.pyro_host
```

Comment:
Is this still something to do with Pyro?

Reply:
Nope, just retaining the existing config without breaking stuff, we can migrate this in the followup PR when the pyro is completely killed